modules/tinymce/src/core/main/ts/api/util/VK.ts

```diff
@@ -46,14 +49,16 @@ const VK: VK = {
   PAGE_DOWN: 34,
   END: 35,
   HOME: 36,
+  CTRL: 17,
```

Comment:
I wonder if we need to document these in the changelog or is it such a fringe api thing that we don't care. What do you guys think? We could also have a set of internal constants but I do see this as being useful in say plugins. At least until we migrate everything to the modern `core` property where everything is strings.

Reply:
I really don't think we should even be including these here... at the very least the information here isn't correct as there's a left/right key for each of these that needs to be accounted for both here and in the actual fix. In addition to that, the meta keycode can actually be different values (e.g. on Chrome its 91/93 and on Firefox it's always 224).

Reply:
I should add that, yes if we are leaving these here then we'll need a changelog entry for this change.

Reply:
This change was removed. Should this discussion be resolved then ?